feature/amazon/build.gradle

```diff
@@ -75,8 +74,6 @@ task cleanAmazonLibrary(type: Delete) {
 }
 
 afterEvaluate {
-    preUnityIAPDebugBuild.dependsOn getAmazonLibrary
-    preUnityIAPReleaseBuild.dependsOn getAmazonLibrary
-    preLatestDependenciesDebugBuild.dependsOn cleanAmazonLibrary
-    preLatestDependenciesReleaseBuild.dependsOn cleanAmazonLibrary
+    preDebugBuild.dependsOn cleanAmazonLibrary
+    preReleaseBuild.dependsOn cleanAmazonLibrary
 }
```

Comment:
We can remove this whole block I think

Reply:
hmm isn't it possible someone could be updating from a version with the amazon library pulled in?

Reply:
This won't run on devs apps, since they get the built code. I think we had this in case we changed between the unityIAP and the latestDependencies. Which makes me think maybe it would be useful to have it for a while, since we will still work on unityIAP at least until we release this to the wild

Reply:
ohhhhh duhhhhh 🤦‍♀️ yeahh, agreed let's leave it. i could add a TODO to remove after unity updates...?

Reply:
sure, but it has minimal impact so I think we are good